src/ansys/dpf/post/helpers/__init__.py

```diff
@@ -0,0 +1,6 @@
+"""Tools package.
+
+Tools
```

Comment:
Helpers?

Reply:
Is there a plan for follow up PR adding an example for streamlines?

Reply:
> The only problem I see is naming. For me "helpers" means wrapping already existing capabilities in another way to make them easier to be used

IMO this is what this method is? The end-goal is to have a much more thorough streamlines manipulation capability, this method being a quick way to plot the streamlines given a Dataframe.



> Is there a plan for follow up PR adding an example for streamlines?

Yes of course, I initially wanted this merged before last Friday's deadline, but if the deadline is not an issue anymore, I can add and example within this PR.

Reply:
> The only problem I see is naming. For me "helpers" means wrapping already existing capabilities in another way to make them easier to be used

@cbellot000 What I can do though is rename `helpers` as `tools`, but that will require deleting the `tools.py` module, which I think can be done without retro-compatibility issues if I import the `select` method from within the `__init__.py` file of the new `tools` sub-package, yet I may have missed a case where it breaks.

Reply:
@PProfizi no, it's okay, your justification makes sense